### PR TITLE
Adds ability to created stacked area charts by setting `stackgroup`

### PR DESCRIPTION
**Pull request by mscottford**
See https://plotly.com/python/filled-area-plots/#stacked-area-chart for more information on `stackgroup`.

Warning: This is my first time working with F#, so I'm almost positive that I got something wrong.

### CODE REVIEW

**Review by cartermp — APPROVED** [review]
Thanks! Tried it out in codepen (the inline JS) it it gives a good graph. Code looks good.

**Comment by mscottford** [comment]
Awesome! Thanks a ton. Any thoughts about when the next release is going to be? How can I help?

**Comment by cartermp** [comment]
Probably sometime soon, after #145 is merged.

I was thinking of holding off until a .NET Interactive extensibility API implementation was added here, but it wouldn't hurt to push out a minor fix

**Pull request merged**
